src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -70,12 +72,16 @@ object SharedFileSystem {
       Failure(new UnsupportedOperationException("Cannot localize directory with symbolic links"))
     else Try(Files.createSymbolicLink(executionPath, originalPath.toAbsolutePath))
   }
+
+  val sharedFSFileHasher: FileHasher = { wdlFile: WdlFile => new io.File(wdlFile.value).md5Sum }
```

Comment:
I wonder if we can use the new "better files" library that Khalid added to do this instead of `io.File().md5sum`.  Or maybe `Path` instead of `File`?  If that's kinda ugly or annoying to change, I think it's fine the way it is